Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM cnstark/pytorch:2.0.1-py3.9.17-cuda11.8.0-ubuntu20.04
```

Comment:
I chose an unofficial base image because it creates smaller Docker images than the official one, at only 30% of its size. Nevertheless, using the official images is always a best practice. I am attempting to recreate the image using an official one, but have encountered some compatibility problems which require additional time and effort.